cmake/thisdd4hep_package.sh.in

```diff
@@ -30,7 +30,7 @@ dd4hep_add_path    PYTHONPATH ${THIS}/@DD4HEP_PYTHON_INSTALL_DIR@;
 #----ROOT_INCLUDE_PATH--------------------------------------------------------
 dd4hep_add_path    ROOT_INCLUDE_PATH ${THIS}/include;
 #----LIBRARY_PATH-------------------------------------------------------------
-dd4hep_add_library_path ${THIS}/lib;
+dd4hep_add_library_path ${THIS}/@CMAKE_INSTALL_LIBDIR@;
```

Comment:
We need to make sue that CMAKE_INSTALL_LIBDIR is set for other packages, which is not guaranteed at the moment.